client/ayon_core/hosts/houdini/plugins/create/create_bgeo.py

```diff
@@ -61,45 +60,56 @@ def create(self, product_name, instance_data, pre_create_data):
 
         instance_node.setParms(parms)
 
+    @staticmethod
+    def update_node_parameters(node, creator_attributes):
+        """update node parameters according to creator attributes.
+
+        Implementation of update_node_parameters.
+
+        Args:
+            node(hou.Node): Houdini node to apply changes to.
+            creator_attributes(dict): Dictionary of creator attributes.
+        """
+
+        file_path, _ = lib.splitext(
+            node.evalParm("sopoutput"),
+            allowed_multidot_extensions=[
+                ".ass.gz", ".bgeo.sc", ".bgeo.gz",
+                ".bgeo.lzma", ".bgeo.bz2"
+            ]
+        )
+
+        output = "{file_path}.{ext}".format(
+            file_path=file_path,
+            ext=creator_attributes["bgeo_type"]
```

Comment:
You are calling this method with the `creator_attributs` from the `new_values` as such I suspect `bgeo_type` might NOT be available if its not changed and this will error?